packages/wallet/src/redux/sagas/messaging/message-handler.ts

```diff
@@ -282,7 +282,7 @@ function* handleUpdateChannelMessage(payload: RequestObject) {
     if (
       protocolState &&
       protocolState.type === "Application.WaitForDispute" &&
-      typeof protocolState.disputeState! !== "undefined"
+      typeof protocolState.disputeState !== "undefined"
```

Comment:
```suggestion
      protocolState.disputeState
```
I am aware this is not exactly equivalent code, but am wondering if it is actually as good or better for our needs here?